src/inputs/Checkbox.tsx

```diff
@@ -2,9 +2,8 @@ import ClickableTooltip from '../ClickableTooltip';
 
 
 function Checkbox(
-  { className, value, name, label, title, setValue, withTooltip}
-  : { className: string,
-      value: boolean,
+  { value, name, label, title, setValue, withTooltip}
+  : { value: boolean,
```

Comment:
It makes sense to move the consistent styles to the component instead of a manually-added class, but I would leave in the _ability_ to pass additional class names for future needs.

Reply:
I feel like I want consistency? So, all or none of the inputs should have a passed in class